src/daemons/MetaDaemon.cpp

```diff
@@ -115,6 +116,8 @@ int main(int argc, char *argv[]) {
             nebula::kvstore::KVStore::instance(std::move(options)));
 
     auto handler = std::make_shared<nebula::meta::MetaServiceHandler>(kvstore.get());
+    auto activeHostsMan = nebula::meta::ActiveHostsMan::instance(kvstore.get());
+    activeHostsMan->loadHostMap();
```

Comment:
We'd better move "activeHostsMan->loadHostMap()" inside instance(),   and use std::call_once to to init the host map at the first time.

Reply:
But loadHostMap use other non-static members, maybe call_once won't work.